install.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# By default, this script will install the latest version of run to $HOME/.local/bin.
+install_dir="$HOME/.local/bin"
+version="latest"
+
+# Get the actual version from the latest release
+if [ "$version" = "latest" ]; then
+    version=$(curl -s https://api.github.com/repos/amonks/run/releases/latest | grep tag_name | cut -d '"' -f 4)
```

Comment:
tradeoff: 
- hitting this API makes the script more complex
- generating the script with a hardcoded version number would make our release process more complex

I don't have a strong opinion either way here; just acknowledging the tension.